deployments/polygon/usdc/migrations/1675200105_configurate_and_ens.ts

```diff
@@ -0,0 +1,262 @@
+
+const cUSDTAddress = '0xf650c3d88d12db855b8bf7d11be6c55a4e07dcc9';
+
+export default migration('1675200105_configurate_and_ens', {
+  prepare: async (deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (deploymentManager: DeploymentManager, govDeploymentManager: DeploymentManager) => {
+    const trace = deploymentManager.tracer();
+    const ethers = deploymentManager.hre.ethers;
+    const { utils } = ethers;
+
+    const {
+      bridgeReceiver,
+      comet,
+      cometAdmin,
+      configurator,
+      rewards,
+      WBTC,
+      WETH,
+      WMATIC
+    } = await deploymentManager.getContracts();
+
+    const {
+      fxRoot,
+      timelock,
+      governor,
+      comptrollerV2,
+      USDC,
+      COMP,
+    } = await govDeploymentManager.getContracts();
+
+    // ENS Setup
+    // See also: https://docs.ens.domains/contract-api-reference/name-processing
+    const ENSResolver = await govDeploymentManager.existing('ENSResolver', ENSResolverAddress);
+    const nameHash = ethers.utils.namehash(ENSName);
+    const labelHash = ethers.utils.keccak256(ethers.utils.toUtf8Bytes(ENSSubdomainLabel));
+    const subdomainHash = ethers.utils.namehash(ENSSubdomain), ttl = 0;
+    const officialMarkets = {
+      1: [
+        {
+          baseSymbol: 'USDC',
+          cometAddress: (await govDeploymentManager.fromDep('cUSDCv3', 'mainnet', 'usdc', 'comet')).address,
+        },
+        {
+          baseSymbol: 'WETH',
+          cometAddress: (await govDeploymentManager.fromDep('cWETHv3', 'mainnet', 'weth', 'comet')).address
+        }
+      ],
+
+      137: [
+        {
+          baseSymbol: 'USDC',
+          cometAddress: comet.address,
+        }
+      ],
+    };
+
+    const configuration = await getConfigurationStruct(deploymentManager);
+
+    const setConfigurationCalldata = await calldata(
+      configurator.populateTransaction.setConfiguration(comet.address, configuration)
+    );
+    const deployAndUpgradeToCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [configurator.address, comet.address]
+    );
+    const l2ProposalData = utils.defaultAbiCoder.encode(
+      ['address[]', 'uint256[]', 'string[]', 'bytes[]'],
+      [
+        [configurator.address, cometAdmin.address],
+        [0, 0],
+        [
+          'setConfiguration(address,(address,address,address,address,address,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint104,uint104,uint104,(address,address,uint8,uint64,uint64,uint64,uint128)[]))',
+          'deployAndUpgradeTo(address,address)'
+        ],
+        [setConfigurationCalldata, deployAndUpgradeToCalldata]
+      ]
+    );
+
+    const RootChainManager = await deploymentManager.existing(
+      'RootChainManager',
+      RootChainManagerAddress
+    );
+    const USDCAmountToBridge = exp(400_000, 6); // roughly half of what's currently in the Timelock
+    const COMPAmountToBridge = exp(25_000, 18);
```

Comment:
I thought we had updated these amounts. Did the rebase remove those changes?

Reply:
Opened a PR here: https://github.com/compound-finance/comet/pull/697

I think we just never updated this after agreeing on the amounts offline 😅

Reply:
Merged into this branch.